sw/device/lib/aes.c

```diff
@@ -101,7 +105,7 @@ void aes_clear(void) {
   }
 
   // Disable autostart
-  REG32(AES_CTRL(0)) = 0x1u << AES_CTRL_MANUAL_OPERATION;
+  REG32(AES_CTRL_SHADOWED(0)) = 0x1u << AES_CTRL_SHADOWED_MANUAL_OPERATION;
```

Comment:
This write needs duplicating, surely?

Reply:
Oh thanks, good catch Sam! I just fixed it.